WIMSE Draft Charter.md

```diff
@@ -6,9 +6,9 @@ The Workload Identity in Multi-Service Environments (WIMSE) working group is cha
 
 # Background and Motivation
 
-The rise of diverse service platforms and the drive for business flexibility, cost-efficiency, resilience, and compliance make maintaining least privilege access for workloads increasingly complex. As a result of the adoption of micro-service architectures, services are composed of multiple workloads that need to authenticate to each other while making authorization decisions based on the original caller, their context, and the actions of other workloads that acted on a transaction. These workloads are often distributed across trust boundaries, without a single centralized controller managing the different identities or authorization policies.
+The drive for business flexibility, cost-efficiency, resilience, and compliance make maintaining least privilege access for services increasingly complex. As a result of the adoption of microservice architectures, applications are composed of multiple microservices that need to authenticate to each other while making authorization decisions based on the original caller, their context, and the actions of other workloads that acted on a request. These microservices are often distributed across trust boundaries, without a single centralized controller managing the different identities or authorization policies.
```

Comment:
I'm thinking we don't want to lose the diverse platform aspect here, since that's a motivating factor for needing interoperability. How about we add it back in later:

```suggestion
The drive for business flexibility, cost-efficiency, resilience, and compliance make maintaining least privilege access for services increasingly complex. As a result of the adoption of microservice architectures, applications are composed of multiple microservices that need to authenticate to each other while making authorization decisions based on the original caller, their context, and the actions of other workloads that acted on a request. These microservices are often distributed across trust boundaries, without a single centralized controller managing the different identities or authorization policies. The diverse platforms for running these services provide a particular interoperability challenge to applications deployed in this way.
```

I'll let others chime in about the change from "workload" -- I'm not married to that term vs service but it's the "W" in "WIMSE" :)

Reply:
@dfeldman can you elaborate more on your thoughts of service vs micro-service vs workload? I can think of some ways to do the wording, but am curious on the thought process behind the suggestion to change from workload to service.